src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -45,20 +53,51 @@ public void periodic() {
       SmartDashboard.putNumber("target Id", table.getEntry("tid").getDouble(0.0));
       SmartDashboard.putNumber("pipe line", pipeln.getDouble(0.0));
       SmartDashboard.putNumber("distance constantly updated", getDistance());
+      System.out.println(getDistance() + "sees a target!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
     }
     // TODO get tgt ID  - based of pipeline
     else {
       SmartDashboard.putBoolean("Target Seen", false);
       SmartDashboard.putNumber("target Id", -99);
       SmartDashboard.putNumber("pipe line", -99);
     }
+
+    // SmartDashboard.putNumber("april tag
+    // id",NetworkTableInstance.getDefault().getTable("limelight").getEntry("tid").getDouble(-10000));
+
+    SmartDashboard.putNumber("distance constantly updated", getDistance());
+
+    pipelineChooser.setDefaultOption(
+        "Red Tag 1", 0); // TODO add the other tags that are on the field
+    pipelineChooser.setDefaultOption("Red Tag 2", 1);
+    pipelineChooser.setDefaultOption("Red Tag 3", 2);
+    pipelineChooser.setDefaultOption("Red Loading Zone Tag 5", 5);
+
+    pipelineChooser.setDefaultOption("Blue Tag 6", 4);
+    pipelineChooser.setDefaultOption("Blue Tag 7", 6);
+    pipelineChooser.setDefaultOption("Blue Tag 8", 7);
+    pipelineChooser.setDefaultOption("Blue Loading Zone Tag 4", 5);
+
+    SmartDashboard.putData("Choose Pipeline", pipelineChooser);
+
+    pLine = pipelineChooser.getSelected().intValue();
+
+    NetworkTableInstance.getDefault().getTable("limelight").getEntry("pipeline").setNumber(pLine);
+
+    if (pLine == 5) {
+      tgtHeightInches = 27.375; // should be 27.375
+      System.out.println("changing target height&&&&&&&&&&&&&&&&&&&&&&&&&");
+
+    } else {
+      tgtHeightInches = 18.0;
+    }
   }
 
   public double getDistance() {
-    double tgtHeightInches = 18.00;
-    double cameraHeightInches = 10.5;
+    // double tgtHeightInches = 18.00;
+    double cameraHeightInches = 10.0;
     double offSetInches = tgtHeightInches - cameraHeightInches;
-    double cameraMountAngleDegrees = -1.0;
+    double cameraMountAngleDegrees = 0.79; // was -1
```

Comment:
Fwiw, these comments aren't necessarily since we can always see the changed value history in git.